docs/reference/release-notes/8.0.0-beta1.asciidoc

```diff
@@ -24,6 +24,17 @@ Also see <<breaking-changes-8.0,Breaking changes in 8.0>>.
   anomaly detection results for different jobs to be mixed.
   Wait for `8.0.0-rc1` before attempting to test upgrading a cluster
   where {ml} was used prior to version `7.7.0`.
+* System indices such as `.security` or `.kibana` are not hidden
+  indices, but they will be in the GA release. As a consequence,
+  these will be included in wildcard expansions for users that
+  have privileges to view them. Do not rely on this behavior; instead,
+  if you need to access a system index, do so directly, or use
+  a more specific wildcard.
```

Comment:
```suggestion
  if you need to access a system index, do so directly, use
  the `expand_wildcards` parameter to request hidden indices.
```

Because I think that's probably the clearer way to specify that you want hidden indices. Alternatively, we could do:

```suggestion
  if you need to access a system index, do so directly, use
  the `expand_wildcards` parameter to request hidden indices,
  or use a pattern which includes the dot prefix, such as `.kib*`.
```

Reply:
I believe the revision addresses this comment.